runtime/defaults/src/main/resources/application.properties

```diff
@@ -68,9 +68,16 @@ quarkus.log.file.path=./logs/polaris.log
 quarkus.log.file.rotation.file-suffix=.yyyy-MM-dd.gz
 quarkus.log.file.rotation.max-file-size=10M
 quarkus.log.file.rotation.max-backup-index=14
-quarkus.log.category."org.apache.polaris".level=INFO
-quarkus.log.category."org.apache.iceberg.rest".level=INFO
-quarkus.log.category."io.smallrye.config".level=INFO
+
+# Useful loggers. Set "io.smallrye.config"to DEBUG to print the resolved configuration properties.
```

Comment:
These loggers were getting in the way for integration tests, so better to comment them out.